extension/lib/webcompat.js

```diff
@@ -81,6 +82,25 @@ class WebCompat {
     return this._toCSSIssues(normalSummaries.concat(aliasSummaries));
   }
 
+  /**
+   * @param {String} elementName -
+   *                e.g. div, main, aside
+   * @param {Array} browsers -
+   *                e.g. [{ id: "firefox", name: "Firefox", version: "68" }, ...]
+   * @return {Object} if no issue found, return null.
+   */
+  getHTMLElementIssue(elementName, browsers) {
+    const database = this._webCompatData.html.elements;
+    const summary = this._getCompatSummary(browsers, database, elementName);
```

Comment:
We might need to do some filtering for custom elements. Go to youtube.com for a good example of why this is important.
They use a lot of custom elements like `ytd-button-renderer` and others.
Currently the extension does not handle this well and fires a warning for those. It should not.

I think you can do this to check: `window.customElements.get(element.localName)`, where `window` is the content page window object.

Reply:
Thank you for catching this!
Yeah, indeed. Will address it.